connectors/rpa/element-templates/rpa-connector.json

```diff
@@ -0,0 +1,145 @@
+{
+  "$schema": "https://unpkg.com/@camunda/zeebe-element-templates-json-schema/resources/schema.json",
+  "name": "RPA Connector",
+  "description": "Execute a Script on the Camunda RPA Worker",
+  "id": "camunda.connectors.rpa",
+  "appliesTo": [
+    "bpmn:Task"
+  ],
+  "category" : {
+    "id" : "connectors",
+    "name" : "Connectors"
+  },
+  "engines": {
+    "camunda": "^8.7"
+  },
+  "elementType": {
+    "value": "bpmn:ServiceTask"
+  },
+  "entriesVisible": false,
+  "groups": [
+    {
+      "id": "worker",
+      "label": "Worker"
+    },
+    {
+      "id": "script",
+      "label": "Script"
+    },
+    {
+      "id": "input",
+      "label": "Input"
+    }
+  ],
+  "properties": [
+    {
+      "type": "Hidden",
+      "binding": {
+        "type": "zeebe:linkedResource",
+        "property": "resourceType",
+        "linkName": "RPAScript"
+      },
+      "value": "RPA"
+    },
+    {
+      "group": "script",
+      "type": "String",
+      "feel": "optional",
+      "label": "Script ID",
+      "binding": {
+        "type": "zeebe:linkedResource",
+        "linkName": "RPAScript",
+        "property": "resourceId"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "group": "script",
+      "type": "Dropdown",
+      "label": "Binding",
+      "id": "bindingType",
+      "binding": {
+        "type": "zeebe:linkedResource",
+        "linkName": "RPAScript",
+        "property": "bindingType"
+      },
+      "choices": [
+        {
+          "name": "Latest",
+          "value": "latest"
+        },
+        {
+          "name": "Deployment",
+          "value": "deployment"
+        },
+        {
+          "name": "Version Tag",
+          "value": "versionTag"
+        }
+      ],
+      "value": "latest"
+    },
+    {
+      "group": "script",
+      "type": "String",
+      "label": "Version tag",
+      "binding": {
+        "type": "zeebe:linkedResource",
+        "linkName": "RPAScript",
+        "property": "versionTag"
+      },
+      "condition": {
+        "property": "bindingType",
+        "equals": "versionTag"
+      }
+    },
+        {
+      "group": "worker",
+      "label": "Label",
+      "tooltip": "Define on which worker this task should be executed.",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "camundaRpaWorkerLabel"
+      },
+      "type": "String",
+      "value": "default",
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "group": "worker",
+      "label": "Timeout",
+      "description": "Timeout of the Script in seconds.",
```

Comment:
It's an ISO-8601 string, need to adjust

Reply:
fixed in [f89e610](https://github.com/camunda/connectors/pull/3949/commits/f89e61052586a697a56bcc1427462edc1b2d605f)